pcap/can_socketcan.h

```diff
@@ -41,14 +41,19 @@
 
 #include <pcap/pcap-inttypes.h>
 
+#define CAN_MTU     16
+#define CANFD_MTU   72
+
+#define CANFD_FDF   0x04 /* mark CAN FD for dual use of CAN format */
```

Comment:
This file is also for the benefit of programs reading frames, so it should also define `CANFD_BRS` and `CANFD_ESI`.